doc/man3/OSSL_STORE_INFO.pod

```diff
@@ -84,15 +84,17 @@ OSSL_STORE_INFO_get0_NAME(), OSSL_STORE_INFO_get0_NAME_description(),
 OSSL_STORE_INFO_get0_PARAMS(), OSSL_STORE_INFO_get0_PUBKEY(),
 OSSL_STORE_INFO_get0_PKEY(), OSSL_STORE_INFO_get0_CERT(),
 OSSL_STORE_INFO_get0_CRL()
-all take a B<OSSL_STORE_INFO> and return the held object of the appropriate
-OpenSSL type provided that's what's held.
+all take a B<OSSL_STORE_INFO> and return the object if holds if the
+B<OSSL_STORE_INFO> type (as returned by OSSL_STORE_INFO_get_type())
+matches the function, otherwise NULL.
 
 OSSL_STORE_INFO_get1_NAME(), OSSL_STORE_INFO_get1_NAME_description(),
 OSSL_STORE_INFO_get1_PARAMS(), OSSL_STORE_INFO_get1_PUBKEY(),
 OSSL_STORE_INFO_get1_PKEY(), OSSL_STORE_INFO_get1_CERT() and
 OSSL_STORE_INFO_get1_CRL()
-all take a B<OSSL_STORE_INFO> and return a duplicate of the held object of the
-appropriate OpenSSL type provided that's what's held.
+all take a B<OSSL_STORE_INFO> and return a duplicate the object if
```

Comment:
if holds if the

makes no sense :)

Reply:
Ok, so please help me out.

For example, `OSSL_STORE_INFO_get1_PUBKEY()` will return non-NULL if `OSSL_STORE_INFO_get_type()` returns `OSSL_STORE_INFO_PUBKEY`, otherwise NULL (even if the **OSSL_STORE_INFO** holds on object, just of a different type).

Reply:
Ohhhhhh, hold on, I didn't quite read what you wrote.  Should be "it holds it", of course...

Reply:
Fixed